### PR TITLE
Work01

### PR DESCRIPTION
**Pull request by madgb**
-css: 95% done
-Webpack, mongo environment ready
-todo: React code split(components) as well as the css files.

### CODE REVIEW

File: db/schema.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+			other: {type: String}
+		},
+		spacesAndMenities:{
+			size: {type: Number},
+			amenities: {type: String},
+			spaces: {type: String}
+		},
+		construction:{
+			typeAndStyle: {type: String},
+			dates:{type:  }{
```
Comment (madgb) [inline db/schema.js]:
```suggestion
			dates:{
```

File: db/schema.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+				remodel: {type: Date} 
+			},
+			materials:{
+				roof: {type: String},
+				exterior: {type: String}
+			},
+			other: {type: String}
+		},
+		exteriorFeatures:{
+			lot:{type:  }{
```
Comment (madgb) [inline db/schema.js]:
```suggestion
			lot:{
```

File: db/index.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+				soldPrice: Number
+			}
+		},
+		activityOnZillow:{
+			daysOnZillow: Date,
+			pastThirtydayView: Number,
+			savedLog: Number
+		}
+	}
+})
```
Comment (b0char0va) [inline db/index.js]: There are some missing semicolon.

File: server/index.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+
+
+const port = process.env.PORT || 8080;
+app.listen(port, ()=>{
+	console.log('working on ', port);
+})
```
Comment (b0char0va) [inline server/index.js]: Extra white space can be removed. Missing semi colon after app.listen.

File: server/routes/index.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+module.exports = (app) => {
+	app.get('/api/details', (req, res) => {
+		res.end();
+	})
+
+	app.get('api/details/:item-id', (req, res) => {
+		res.end();
+	})
```
Comment (b0char0va) [inline server/routes/index.js]: missing semi colons

File: client/dist/index.html [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (b0char0va) [inline client/dist/index.html:3]: can get rid of white space here

File: client/dist/scss/_cal.scss [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+	min-width: 300px;
+	border: 1px solid #aaa;
+	box-shadow: 0 0 2px #444;
+	z-index: 900;
+	.cal-title{
+		text-align: center;
+		padding-top: 20px;
+	}
+	.cal-body{
+    background-color: #cdf2db;
```
Comment (b0char0va) [inline client/dist/scss/_cal.scss]: seems like not indented properly

**Pull request merged**